@xen-orchestra/backups/_runners/VmsRemote.mjs

```diff
@@ -77,16 +86,33 @@ export const VmsRemote = class RemoteVmsBackupRunner extends Abstract {
             throttleStream,
             vmUuid,
           }
-          let vmBackup
-          if (job.mode === 'delta') {
-            vmBackup = new IncrementalRemote(opts)
-          } else if (job.mode === 'full') {
-            vmBackup = new FullRemote(opts)
-          } else {
-            throw new Error(`Job mode ${job.mode} not implemented for mirror backup`)
-          }
 
-          return runTask(taskStart, () => vmBackup.run())
+          return runTask(taskStart, () => {
+            return pRetry(
+              () => {
+                let vmBackup
+                switch (job.mode) {
+                  case 'delta':
+                    vmBackup = new IncrementalRemote(opts)
+                    break
+                  case 'full':
+                    vmBackup = new FullRemote(opts)
+                    break
+                  default:
+                    throw notImplementedJobModeError
+                }
+
+                return vmBackup.run()
+              },
+              {
+                delays: fibonacci()
+                  .take(allSettings[''].nRetriesVmBackupFailures ?? 0)
+                  .toMs(),
+                onRetry: () => debug('retry mirror backup job', { jobId: job.id }),
```

Comment:
since a retry is quite important, I would show in the the UI, with a Task.warning, that way the users will be able to see a failing process